property_caching/__init__.py

```diff
@@ -38,12 +45,14 @@ def _get_cache(obj, cache_attr_name=_OBJ_CACHE_ATTR_NAME):
     return getattr(obj, cache_attr_name, {})
 
 
-def _get_property_value(fn, obj, cache_attr_name):
+def _get_property_value(fn, obj, cache_attr_name, only_cache_true_vals=False):
```

Comment:
What about `cache_false_results` (and flipping the logic)? It's shorter and reads the same as your example.